manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,96 @@
+class icinga2::pnp4nagios::config {
+
+  $package_name           = $icinga2::pnp4nagios::package_name
+  $package_ensure         = $icinga2::pnp4nagios::package_ensure
+  $service_name           = $icinga2::pnp4nagios::service_name
+  $service_ensure         = $icinga2::pnp4nagios::service_ensure
+  $service_enable         = $icinga2::pnp4nagios::service_enable
+  $service_manage         = $icinga2::pnp4nagios::service_manage
+  $htpasswd_config        = $icinga2::pnp4nagios::htpasswd_config
+  $htpasswd_user          = $icinga2::pnp4nagios::htpasswd_user
+  $htpasswd_group         = $icinga2::pnp4nagios::htpasswd_group
+  $htpasswd_config_ensure = $icinga2::pnp4nagios::htpasswd_config_ensure
+  $htpasswd_config_mode   = $icinga2::pnp4nagios::htpasswd_config_mode
+  $httpd_path             = $icinga2::pnp4nagios::httpd_path
+  $nagios_web_user        = $icinga2::pnp4nagios::nagios_web_user
+  $nagios_web_pass        = $icinga2::pnp4nagios::nagios_web_pass
+  $manage_pkg_dependency  = $icinga2::pnp4nagios::manage_pkg_dependency
+  $monitoring_type        = $icinga2::pnp4nagios::monitoring_type
+  $log_type               = $icinga2::pnp4nagios::log_type
+  $debug_lvl              = $icinga2::pnp4nagios::debug_lvl
+  $system_date            = $icinga2::pnp4nagios::system_date 
+
+  file { "$httpd_path/$htpasswd_config":
+    ensure  => $htpasswd_config_ensure,
+    owner   => $htpasswd_user,
+    group   => $htpasswd_group,
+    mode    => $htpasswd_config_mode,
+    content => template('icinga2/pnp4nagios.conf.erb'),
+    notify  => Service['httpd'],
+  }
+
+  file { '/etc/nagios/passwd':
+    ensure  => file,
+    owner   => $htpasswd_user,
+    group   => $htpasswd_group,
+    notify  => Service['httpd'],
+  }
+
+  ini_subsetting { "nagiospasswd":
+    ensure  => present,
+    section           => '',
+    key_val_separator => ':',
+    path              => '/etc/nagios/passwd',
+    setting           => $nagios_web_user,
+    subsetting        => '',
+    value             => $nagios_web_pass,
+  }
+
+  file { '/etc/pnp4nagios/npcd.cfg':
+    owner   => $htpasswd_user,
+    group   => $htpasswd_group,
+    notify  => Class[icinga2::pnp4nagios::service],
+    content => template('icinga2/npcd.cfg.erb'),
+  }
+
+  ini_setting { "phptimezone":
```

Comment:
I remember us talking about an if statement around this with a class parameter $manage_php_timezone?